Library/Homebrew/test/support/lib/startup/config.rb

```diff
@@ -15,6 +16,13 @@
   ENV[k] = dir
 end.freeze
 
+# Use a shorter HOMEBREW_TEMP path so Sequoia doesn't error out as often on long paths (> 104 bytes).
+# Use the minimal amount of randomness to avoid collisions while allowing parallel tests.
+require "securerandom"
+random_hex = SecureRandom.hex(1)
+HOMEBREW_TEMP = Pathname("#{homebrew_temp}/brewtests#{random_hex}".squeeze("/")).freeze
```

Comment:
@carlocab Was working fine locally with 10 workers without any `random_hex` at all. Would rather keep the `tests` identifier in here for now but this will be the first approach we can jump to if we have any problems.

Reply:
Note: this produces two characters:
```
brew(main):004> SecureRandom.hex(1)
=> "9b"
```